lib/electric_slide/agent.rb

```diff
@@ -2,7 +2,7 @@
 
 class ElectricSlide
   class Agent
-    attr_accessor :id, :address, :presence, :call, :queue
+    attr_accessor :id, :address, :presence, :call, :queue, :removed_by
```

Comment:
You mean doing something like:

`agent.presence = :unavailable, removed_by: 'agent_id'`